### ISSUE TITLE
Daemon using CRDTs fails to start on Raspberry Pi

### ISSUE THREAD

**Issue by lanzafame** [ipfs/ipfs-cluster#771]
**Additional information:**
 - OS: linux
 - IPFS Cluster version: master
 - Installation method: built from source for ARM

Running ipfs-cluster-service daemon on a Raspberry Pi 3 B with CRDT consensus fails due to badger not being able to `mmap` memory. 
```
$ ipfs-cluster-service daemon --consensus=crdt
06:07:40.790  INFO    service: Initializing. For verbose output run with "-l debug". Please wait... daemon.go:48
error creating datastore: Mmap value log file. Path=/home/pi/.ipfs-cluster/badger/000000.vlog. Error=cannot allocate memory
```

I found this [badgerdb issue](https://github.com/dgraph-io/badger/issues/610) which is resolved and if I ran that main.go on the RPi, it worked but if I changed the `options.ValueLogLoadingMode` back to the default, which is to use `options.MemoryMap`, that example program fails. 

@hsanjuan when you get the chance could you please confirm this? 

The fix, if this is a problem, is to allow passing through the configuration of value for `options.ValueLogLoadingMode` to badger.

**Comment by hsanjuan** [comment]
@lanzafame `ValueLogFileSize` is set to 1GB by default. Can you try reducing it? How much RAM is there in these Raspberry Pis ? Ideally we would automatically handle this in systems with lower memory somehow. We can always expose [needed] badger options though.

**Comment by hsanjuan** [comment]
Closed by #776 and configuring file loading mode for badger.
